manager/allocator/cnmallocator/drivers_ipam.go

```diff
@@ -1,14 +1,39 @@
 package cnmallocator
 
 import (
+	"net"
+	"strconv"
+	"strings"
+
 	"github.com/docker/libnetwork/drvregistry"
 	"github.com/docker/libnetwork/ipamapi"
 	builtinIpam "github.com/docker/libnetwork/ipams/builtin"
 	nullIpam "github.com/docker/libnetwork/ipams/null"
 	remoteIpam "github.com/docker/libnetwork/ipams/remote"
+	"github.com/docker/libnetwork/ipamutils"
+	"github.com/sirupsen/logrus"
 )
 
-func initIPAMDrivers(r *drvregistry.DrvRegistry) error {
+func initIPAMDrivers(r *drvregistry.DrvRegistry, defaultAddrPool []*net.IPNet, subnetSize int) error {
+	var addressPool []*ipamutils.NetworkToSplit
+	var str strings.Builder
+	str.WriteString("Subnetlist - ")
+	if defaultAddrPool != nil {
+		for _, p := range defaultAddrPool {
+			// trim leading and trailing white spaces
+			poolsDef := ipamutils.NetworkToSplit{}
+			poolsDef.Base = p.String()
+			poolsDef.Size = subnetSize
+			addressPool = append(addressPool, &poolsDef)
```

Comment:
These 4 lines could be perhaps more cleanly written as:
```
        addressPool = append(addressPool, &ipamutils.NetworkToSplit {
                Base: p.String(),
                Size: subnetSize),
        }
```
Even if not merging the append and declaration lines, I think the initialization format makes things more readable.